Step-by-step-Tutorials/Unit-Testing-Grains.md

```diff
@@ -69,7 +69,7 @@ namespace Tests
 
 ```
 
-As you can see having `ClassCleanup()` is optional but it's good to have it around. 
+Make sure to add `async` to the generated test method for the test to be recognized by Visual Studio. As you can see having `ClassCleanup()` is optional but it's good to have it around. 
```

Comment:
Hi @jdom. I tried to run the tests in Visual Studio and stumble upon this. However, my memory fooled me and the `async` is not the culprit. What I ran into is that if you leave the return type of the method to be `void`, then there is no error message but test does not show up and cannot be executed. Since the test method is pre-generated by VS with `void` as return type, I guess that it is easy to miss (like in my case). Long story short: your sentence is spot on - especially the part that it must return a `Task` solves what I had in mind.